script/prepare-release.js

```diff
@@ -166,12 +173,37 @@ async function runReleaseBuilds () {
   })
 }
 
+async function verifyNewVersion () {
+  let newVersion = await getNewVersion(true)
+  let response = await promptForVersion(newVersion)
+  if (response !== 'Y') {
+    console.log(`${fail} Aborting release of ${newVersion}`)
+    process.exit()
+  } else {
+    console.log(`${pass} Starting release of ${newVersion}`)
+  }
+}
+
+async function promptForVersion (version) {
+  return new Promise((resolve, reject) => {
+    const rl = readline.createInterface({
+      input: process.stdin,
+      output: process.stdout
+    })
+    rl.question(`Do you want to create the release ${version.green} (Y/N)? `, (answer) => {
```

Comment:
This is a little nonidiomatic: generally choices are lowercase and the default choice (if any) is uppercase. So my brain parses this line as "there are two default choices" :confused:

Reply:
@ckerr I updated to (y/N) since No is the default